### ISSUE TITLE
Subsection rendering is broken on master

### ISSUE THREAD

**Issue by Rikuoja** [City-of-Helsinki/kerrokantasi-ui#438]
Yields 

```
SortableCommentList.js:50 Uncaught TypeError: t is not a function
    at e.t (SortableCommentList.js:50)
    at e.value (SortableCommentList.js:55)
    at t.notifyAll (CallbackQueue.js:76)
    at r.close (ReactReconcileTransaction.js:80)
    at r.closeAll (Transaction.js:209)
    at r.perform (Transaction.js:156)
    at o.perform (Transaction.js:143)
    at o.perform (ReactUpdates.js:89)
    at Object.C [as flushBatchedUpdates] (ReactUpdates.js:172)
    at r.closeAll (Transaction.js:209)
    at r.perform (Transaction.js:156)
    at Object.batchedUpdates (ReactDefaultBatchingStrategy.js:62)
    at Object.i [as batchedUpdates] (ReactUpdates.js:97)
    at dispatchEvent (ReactEventListener.js:147)
```

Must look at diff between master and production to find out when this happened.

**Comment by Rikuoja** [comment]
Cannot resolve #392 before the basic bug is fixed.

**Comment by Rikuoja** [comment]
After this is fixed, please proceed to https://github.com/City-of-Helsinki/kerrokantasi-ui/issues/418 . We really need smoke tests that check that 1) all the views render reliably and 2) posting a comment works, otherwise we will end up with the same problems at every refactor.

**Comment by jonnetanninen** [comment]
I can't reproduce this exact error, but I'm pretty sure PR #440 should fix this issue also.

**Comment by Rikuoja** [comment]
Fixed in #440
